OpenDreamShared/Dream/IconAppearance.cs

```diff
@@ -58,6 +69,7 @@ public sealed class IconAppearance : IEquatable<IconAppearance> {
     ];
 
     public IconAppearance() {
+        Name = string.Empty;
```

Comment:
You should follow the pattern and put this in the field definition instead (`public string Name = string.Empty`)